test/integration/host_js_test.rb

```diff
@@ -225,7 +225,7 @@ class HostJSTest < IntegrationTestWithJavascript
       page.find('#check_all').click
 
       # Ensure all hosts are checked
-      assert page.find('input.host_select_boxes').checked?
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
```

Comment:
I've tried to improve the comment above the line to explain what it's waiting for more precisely.